### PR TITLE
Fix pytest.org link in docs

### LINKED ISSUE #13182: Links to pytest.org in docs are broken
### Describe the problem and steps to reproduce it:
1. Visit https://addons-server.readthedocs.io/en/latest/topics/development/testing.html
2. Click the `pytest` link

### What happened?
'Sorry. This page does not exist yet'

### What did you expect to happen?
It links to 'pytest.org' correctly.

### PR DESCRIPTION
**Pull request by mirefly**
Fixes #13182

### CODE REVIEW

File: docs/topics/development/testing.rst [hunk 1]
```diff
@@ -178,9 +178,9 @@ need to recompile the .mo files manually, for example::
 
 .. _`Django's Unit Testing`: http://docs.djangoproject.com/en/dev/topics/testing
 .. _`PyPom`: http://pypom.readthedocs.io/en/latest/
-.. _`pytest`: http://pytest.org/latest/
+.. _`pytest`: http://pytest.org/en/latest/
```
Comment (EnTeQuAk) [inline docs/topics/development/testing.rst]: Let's use only `pytest.org` here instead to help the user land on a page in his personal preferred language.

File: docs/topics/install/deprecated/elasticsearch.rst [hunk 2]
```diff
@@ -104,7 +104,7 @@ or
 
     make test_es
 
-.. _pytest: http://pytest.org/latest/
+.. _pytest: http://pytest.org/en/latest/
```
Comment (EnTeQuAk) [inline docs/topics/install/deprecated/elasticsearch.rst]: Let's only use `pytest.org` here too.

File: docs/topics/development/testing.rst (hunk 1 above)
Comment (mirefly) [reply docs/topics/development/testing.rst]: Thanks @EnTeQuAk I updated this PR.

**Comment by mirefly** [comment]
Filed #13182

**Pull request merged**